ocaml/typing/typemode.ml

```diff
@@ -1,10 +1,6 @@
 open Location
 open Mode
 
-type error = Unrecognized_modality of string
-
-exception Error of Location.t * error
-
 let transl_mode_annots modes =
   Typemodifier.transl_mode_annots ~required_mode_maturity:Stable modes
```

Comment:
I think it's odd to have `transl_mode_annots` in `Typemodifier`. Because there are many places where mode annots is not modifying a type. Most common example is `let x @ local = ..`.

I think `transl_mode_annotes` should stay in `Typemode`.